web/src/test/java/org/openmrs/web/filter/startuperror/StartupErrorFilterTest.java

```diff
@@ -55,20 +59,18 @@ public void getModel_shouldReturnAStartupErrorFilterModelContainingTheStartupErr
 	@Test
 	public void skipFilter_shouldReturnTrueIfNoErrorHasOccuredOnStartup() {
 		
-		when(Listener.errorOccurredAtStartup()).thenReturn(false);
+		Whitebox.setInternalState(Listener.class, "errorAtStartup", false);
 		
-		StartupErrorFilter filter = new StartupErrorFilter();
 		
-		assertTrue("should be true on start without error", filter.skipFilter(new MockHttpServletRequest()));
+		assertTrue(filter.skipFilter(new MockHttpServletRequest()), "should be true on start without error");
 	}
 	
 	@Test
 	public void skipFilter_shouldReturnFalseIfAnErrorHasOccuredOnStartup() {
 		
-		when(Listener.errorOccurredAtStartup()).thenReturn(true);
+		Whitebox.setInternalState(Listener.class, "errorAtStartup", true);
```

Comment:
you will need to set the errors field to a Throwable (for example an Exception) for the getter to return true